scripts/check-file-synchronization.js

```diff
@@ -1,6 +1,4 @@
-const synchronizations = {
-  'pwa-docker-build': ['Dockerfile', 'Dockerfile_noSSR'],
-};
+const synchronizations = {};
```

Comment:
Now that the synchronizations object is empty, this script doesn't do anything anymore, does it?
Are you keeping it for those projects who do need it? Or can we remove it?